<!-- BEGIN KEY -->
public class javax.security.auth.RefreshFailedException extends java.lang.Exception
<!-- END KEY -->
<!-- BEGIN DATA -->
当 <code>refresh</code> 操作失败时，抛出此异常。
 
<p> 此异常由实现 <code>Refreshable</code> 接口的证书在 <code>refresh</code> 方法失败时抛出。

@version 1.8, 12/19/03

<!-- END DATA -->
<!-- BEGIN KEY -->
public javax.security.auth.RefreshFailedException()
<!-- END KEY -->
<!-- BEGIN DATA -->
构造不带详细消息的  RefreshFailedException。详细消息是描述此特定异常的 String。

<!-- END DATA -->
<!-- BEGIN KEY -->
public javax.security.auth.RefreshFailedException(String msg)
<!-- END KEY -->
<!-- BEGIN DATA -->
构造带指定详细消息的 RefreshFailedException。详细消息是描述此特定异常的 String。

 <p>

@param msg 详细消息。  

<!-- END DATA -->
